README.md

```diff
@@ -208,12 +208,12 @@ Active Team
 
 ### Maintainers* and core developers
 
-- Dr. Sergei Bastrakov
+- Dr. Sergei Bastrakov*
 - Dr. Alexander Debus
-- Marco Garten
+- Marco Garten*
 - Axel Huebl*
 - Alexander Matthes
-- Richard Pausch*
+- Dr. Richard Pausch*
```

Comment:
Please wait with that addition until I have the official document

Reply:
Ah, sorry, I am not aware of German rules on that. Will revert.

Reply:
No problem - that's why we review it 😉